code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -276,7 +282,11 @@
 							var/obj/vehicle/multitile/root/cm_armored/A = O
 							A.take_damage_type(rand(40,60) + 8 * upgrade, "acid", src)
 							A.healthcheck()
-						normal_density_flag = 1
+						else if(istype(O, /obj/structure/razorwire) )
```

Comment:
Another snowflake checks that should be handled by components.

Reply:
Well this is currently the norm unfortunately.

Reply:
If this is cause for barring the PR, we should just delete the entire ability.

Reply:
We dont have components yet so this is fine